src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -103,9 +103,8 @@ public void visitJavadocToken(DetailNode ast) {
         if (!isInlineDescription(ast)) {
             final List<DetailNode> textNodes = getAllNewlineNodes(ast);
             for (DetailNode newlineNode : textNodes) {
-                final DetailNode textNode = JavadocUtil.getNextSibling(JavadocUtil
-                        .getNextSibling(newlineNode));
-                if (textNode != null && textNode.getType() == JavadocTokenTypes.TEXT) {
+                final DetailNode textNode = JavadocUtil.getNextSibling(newlineNode);
+                if (textNode.getType() == JavadocTokenTypes.TEXT) {
```

Comment:
Does it matter if line is started with non-text like `<b>` tag or something similar? Can you ensure the tests have an example.

Reply:
Sure I'll add this example to the input of test case

Reply:
@rnveach `JavadocIndentationCheck` does not include HTML_ELEMENT.
So in the example below:
```
1.  /**
2.  * Some description here.
3.  * @param a 
4.  * <b> Bold text </b>
5.  */
6.  public void test() {}

Line number 4 will not throw a violation.
```

Reply:
It should be fixed. Please create a new issue for it.

Reply:
Sure

Reply:
Created an issue for this https://github.com/checkstyle/checkstyle/issues/8183

Reply:
> Please update https://checkstyle.org/config_javadoc.html#JavadocTagContinuationIndentation_Description with description on how logic should behave.

@rnveach Done 👍 , please check if it is clear enough or not.

Reply:
Thanks. This resolves the item.